.buildkite/pipeline.yml

```diff
@@ -233,8 +233,6 @@ steps:
       - set-metadata
     command: ".buildkite/steps/build-debian-packages.sh"
     artifact_paths: "deb/**/*"
-    agents:
-      queue: "deploy"
```

Comment:
Do we not need to target some other agent rule?

Reply:
the elastic-runners stack processes the default queue